### DISCUSSION TITLE
Does current Mastodon API support video upload?

### DISCUSSION THREAD

**Discussion by moniesoke** [mastodon/mastodon#20435] [General]
I am so glad that my Mastodon server is working well. Now the real business starts.

I have been using API of twitter, telegram and YouTube to upload videos in batch. Just wondering whether Mastodon also support this?

Thanks!

**Comment by Gargron** [comment]
Yes

**Reply by moniesoke** [reply]
Could you tell me where it is? I couldn't find it on the documentation page of mastodon. Thanks!

**Reply by MrGeneration** [reply]
Did you have a look on these two documentation pages?
https://docs.joinmastodon.org/entities/attachment/
https://docs.joinmastodon.org/methods/statuses/media/

**Reply by moniesoke** [reply]
Oh, I don't know how I missed this. Thanks.
